src/components/city-weather.js

```diff
@@ -0,0 +1,60 @@
+import { useSelector } from "react-redux"
+import { Sparklines, SparklinesLine,SparklinesReferenceLine } from "react-sparklines"
+
+
+const City_Weather = () => {
+  const city = useSelector((state) => state)
+
+  const dataAvg = (data) =>{
+    if (data === undefined) {
+      return
+    }
+    const lodash = require('lodash')
+    const arr = data
+    const mean = lodash.mean(arr).toFixed(2)
+    return mean 
+  }
+
+    return ( 
+      <div>
+        <div className="row col-sm-10 offset-1 titles">
+          <p className="col-sm-3 name" align="center"><strong>{city.city}</strong></p>
+          <p className="col-sm-3 graph" align="center"> 
+            <Sparklines data={city.temp}>
```

Comment:
This could be more DRY (don't repeat yourself). As things stand, you're effectively repeating the sparklines section three times.